server/tests/oauth.test.js

```diff
@@ -0,0 +1,112 @@
+import passport from 'passport';
+import * as chai from 'chai';
+import sinonChai from 'sinon-chai';
+import sinon from 'sinon';
+import request from 'supertest';
+import { googleProfile, facebookProfile } from './mochData/socialMock';
+import data from './mochData/socialMockData';
+import VerifyUser from '../helpers/verifyUser';
+import CheckUser from '../middlewares/checkUser';
+
+import app from '../index';
+
+chai.use(sinonChai);
+chai.should();
+const { expect } = chai;
+
+
+describe('Social authentication tests', () => {
```

Comment:
Update the comment to **Google Social Authentication Test**.

Reply:
Feedback Implemented!

Reply:
Not yet.